vm/src/randomness.rs

```diff
@@ -1,5 +1,57 @@
 // Copyright 2020 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+use encoding::serde_bytes::{self, ByteBuf};
+use serde::{de, Deserialize, Deserializer, Serialize, Serializer};
+use std::fmt::{self, Debug, Formatter};
+
 /// String of random bytes
-pub type Randomness = [u8; 32];
+#[derive(PartialEq, Eq, Default, Copy, Clone)]
+pub struct Randomness(pub [u8; 32]);
+
+impl Debug for Randomness {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        self.0.fmt(f)
+    }
+}
```

Comment:
if you use this wrapper, we will just have to implement every trait we want to use for it, which gets clunky, probably best to just use a serialize tag to do this logic (as mentioned with the Byte32 removal) instead of unnecessarily having to use this type